persistence-api/src/main/java/io/stargate/db/DbActivator.java

```diff
@@ -0,0 +1,30 @@
+package io.stargate.db;
+
+import io.stargate.core.activator.BaseActivator;
+import io.stargate.db.datastore.DataStoreFactory;
+import java.util.Collections;
+import java.util.List;
+
+public class DbActivator extends BaseActivator {
+  private final ServicePointer<Persistence> persistence =
+      BaseActivator.ServicePointer.create(Persistence.class);
+
+  public DbActivator() {
+    super("DB services");
+  }
+
+  @Override
+  protected ServiceAndProperties createService() {
+    return new ServiceAndProperties(new DataStoreFactory(), DataStoreFactory.class);
+  }
+
+  @Override
+  protected void stopService() {
+    // no-op
+  }
+
+  @Override
+  protected List<ServicePointer<?>> dependencies() {
+    return Collections.singletonList(persistence);
```

Comment:
Why does this activator have to depend on `Persistence` if it does not use the reference to the service apparently?

Reply:
That was a left-over. Good catch removed